target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -12,6 +12,7 @@ ramips_setup_interfaces()
 	alfa-network,awusfree1|\
 	d-team,pbr-d1|\
 	tama,w06|\
+	tplink,re305-v1|\
 	tplink,tl-mr3020-v3|\
 	tplink,tl-wr802n-v4)
 		ucidef_set_interface_lan "eth0"
```

Comment:
One could argue, that we need switch configuration, but as this is missing for other devices also (not only on ramips), we should be fine here. Just something to look out when planning to use VLANs.